fimsauthz-api/fims-authz.yaml

```diff
@@ -337,7 +357,18 @@ components:
             [dss.read.identification_service_areas, rid.read.enhanced_details]
           type: string
           example: "scope1 scope2"
-
+        aud:
+          description: |
+            The "aud" (audience) claim identifies the recipients that the JWT is
+            intended for.  Each principal intended to process the JWT MUST
```

Comment:
Line 12  change "flow for UTM components that are Oauth2 Resources servers" to ... Oauth2 Resource Service or oauth2 resource API.

Reply:
We are introducing the term 'principal' whereas we have 2 usages of oauth resource API/service and dozens usages of  'client'

Reply:
ok... this was a cut-paste from RFI for speed.  I'll edit a bit more to it is more like our other language

Reply:
changed "principal" to "server" in most cases and removed a few other words to make cleaner.